torch/csrc/jit/graph_executor.cpp

```diff
@@ -309,8 +309,12 @@ struct DifferentiableGraphBackward : public autograd::Function {
 // to the output Variables if present.
 struct DifferentiableGraphOp {
   DifferentiableGraphOp(Gradient grad)
-      : f(grad.f),
-        grad(std::move(grad)),
+      : grad(std::move(grad)),
```

Comment:
It would probably be more approriate to define a copy constructor for DifferentiableGraphOp rather than function because this is private.